webcrypto/subtle_crypto.go

```diff
@@ -446,7 +446,7 @@ func (sc *SubtleCrypto) Digest(algorithm sobek.Value, data sobek.Value) *sobek.P
 		var ok bool
 		hashFn, ok = getHashFn(normalized.Name)
 		if !ok {
-			return NewError(NotSupportedError, "unsupported algorithm: "+normalized.Name)
+			return NewError(NotSupportedError, "unsupported algorithm, can't map hash function: "+normalized.Name)
```

Comment:
Nit, as we're at it we could go full explicit and help the user understand how to solve the issue, what do you think of something along the lines of the following?

```suggestion
			return NewError(
				NotSupportedError,
				"unsupported digest algorithm '"+normalized.Name+"', "+
				"accepted values are: SHA-1, SHA-256, SHA-384, and SHA-512",
			)
```